tests/ZendTest/Db/Sql/Predicate/PredicateTest.php

```diff
@@ -227,6 +227,14 @@ public function testExpression()
             array(array('foo = %s', array(0), array(Expression::TYPE_VALUE))),
             $predicate->getExpressionData()
         );
+
+        //test with no parameters
```

Comment:
I think it is better to add a new test instead of adding to existing ones. A new test would be:
```php
public function testExpressionParametersDefaultToArray()
{
    $predicate = new Predicate;
    $this->assertSame($predicate, $predicate->expression('foo = bar'));
    $this->assertEquals(
        array(array('foo = bar', array(), array())),
        $predicate->getExpressionData()
    );
}
```